examples/platform/nrfconnect/util/PigweedLogger.cpp

```diff
@@ -146,4 +146,14 @@ LOG_BACKEND_DEFINE(pigweedLogBackend, pigweedLogApi, /* autostart */ true);
 #endif // CONFIG_LOG
 
 } // namespace
+
```

Comment:
Should we update the year to 2020-2021 here since we are modifying the file?

Reply:
I don't think so, but I am not a lawyer...